projects/Mallard/src/hooks/use-image-size.ts

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react'
+import { ImageSize } from '../../../common/src'
+import { imageForScreenSize } from 'src/helpers/screen'
+
+const useImageSize = () => {
+    const [imageSize, setImageSize] = useState<ImageSize>('phone')
+
+    useEffect(() => {
+        imageForScreenSize().then(imgSize => {
+            setImageSize(imgSize)
+        })
+    })
+
+    return {
+        imageSize,
+    }
+}
+
+export { useImageSize }
```

Comment:
should we try to move this to the top level of the app and propagate it via context so we don't flash in phone-sized images then immediately rerender?

Reply:
I've reduced this now but for one area its being tricky. Keen to get the beta out and look at this from a higher level

Reply:
totes!